k8s/namespaces/nfdiv/nfdiv-cron-process-failed-pronounced-cases/aat-00.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: nfdiv-cron-process-failed-pronounced-cases
+spec:
+  releaseName: nfdiv-cron-process-failed-pronounced-cases
+  values:
+    job:
+      environment:
+        NOTIFY_TEMPLATE_SIGN_IN_DIVORCE_URL: https://nfdiv-apply-for-divorce.aat.platform.hmcts.net/
+        NOTIFY_TEMPLATE_SIGN_IN_DISSOLUTION_URL: https://nfdiv-end-civil-partnership.aat.platform.hmcts.net/
+        NOTIFY_TEMPLATE_APPLICANT2_SIGN_IN_DIVORCE_URL: https://nfdiv-apply-for-divorce.aat.platform.hmcts.net/applicant2
+        NOTIFY_TEMPLATE_APPLICANT2_SIGN_IN_DISSOLUTION_URL: https://nfdiv-end-civil-partnership.aat.platform.hmcts.net/applicant2
+        S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+        FEE_API_URL: http://fees-register-api-aat.service.core-compute-aat.internal
+        IDAM_API_REDIRECT_URL: https://div-pfe-aat.service.core-compute-aat.internal/authenticated
```

Comment:
none of these urls look like they should be here, they should be included in your chart and templates with `.Values.global.environment`

Reply:
@timja we use the same image for application and cron job and hence we are overriding it here

Reply:
or you using we don't need to include it here and they will be applied with what we have defined in our charts here https://github.com/hmcts/nfdiv-case-api/tree/master/charts/nfdiv-case-api ?

Reply:
yeah, it will get overridden for you

Reply:
URLs removed
https://github.com/hmcts/cnp-flux-config/pull/12494/commits/fd7dc175c2d4bf594c0b8b2c95b3addfc284d689